response_operations_ui/controllers/survey_controllers.py

```diff
@@ -32,6 +34,17 @@ def get_survey(short_name):
     return response.json()
 
 
+def fdi_survey_short_name_wrapper(get_survey_short_name_func):
+    """Function wrapper that converts the FDI specific survey names to all return as the string 'FDI'"""
+    @wraps(get_survey_short_name_func)
+    def convert_fdi_surveys(survey_id):
+        survey_short_name = get_survey_short_name_func(survey_id)
+        if survey_short_name in FDISurveys.__members__:
```

Comment:
enum is an iterable so access to __members__ isn't need you can just do:

for survey in FDISurveys: